src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    box-sizing: border-box;
+  }
+
+  &__title {
+    color: $main-accent-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 21px;
+
+    &-text {
+      color: $main-accent-color;
+      text-align: right;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-title {
+      color: $secondary-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-value {
+      color: $main-accent-color;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__buy:link {
+    text-decoration: none;
+    text-transform: uppercase;
+    border: none;
+    color: $text-color;
+    font-weight: 700;
+    line-height: 16px;
+    font-size: 14px;
+    border-radius: 5px;
+    background: $button-color;
+    padding: 12px 70px 12px 69px;
+  }
+
+  &__buy:visited {
+    color: $text-color;
+  }
+
+  &__buy:hover {
+    padding-top: 11px;
+    padding-bottom: 11px;
+    box-sizing: border-box;
+    border: 1px solid $button-color;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover